src/Promitor.Integrations.Sinks.OpenTelemetry/Configuration/OpenTelemetryCollectorSinkConfiguration.cs

```diff
@@ -1,7 +1,16 @@
-﻿namespace Promitor.Integrations.Sinks.OpenTelemetry.Configuration
+﻿using OpenTelemetry.Exporter;
+
+namespace Promitor.Integrations.Sinks.OpenTelemetry.Configuration
 {
     public class OpenTelemetryCollectorSinkConfiguration
     {
         public string CollectorUri { get; set; }
+        
+        public CollectorInfo Collector { get; set; }
+
+        public class CollectorInfo
```

Comment:
Let's move this to a dedicated file

Reply:
I think with the generators etc that it may be best to leave as is? Can you look at the change and verify?